omnibus/config/templates/datadog-agentless-scanner/systemd.service.erb

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Datadog Agentless Scanner
+After=network.target datadog-agent.service
+BindsTo=datadog-agent.service
+ConditionPathExists=<%= etc_dir %>/datadog.yaml
+
+[Service]
+Type=simple
+PIDFile=<%= install_dir %>/run/agentless-scanner.pid
+Restart=on-failure
+ExecStart=<%= install_dir %>/bin/agentless-scanner run -c <%= etc_dir %>/datadog.yaml -p <%= install_dir %>/run/agentless-scanner.pid
+# Since systemd 229, should be in [Unit] but in order to support systemd <229,
```

Comment:
Do we need to care about old systemd versions for a new product?

Reply:
Done.